src/process_utility.c

```diff
@@ -2697,6 +2697,7 @@ process_altertable_end_subcmd(Hypertable *ht, Node *parsetree, ObjectAddress *ob
 			ereport(ERROR,
 					(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
 					 errmsg("hypertables do not support logical replication")));
+			break;
```

Comment:
I am worried that this will lead down a path where we require a `pg_uncreachable` after every error statement. There are lots of places where we have breaks and returns after ereport(ERROR) and don't have `pg_uncreachable`.